docs/dictionary-cache.md

```diff
@@ -0,0 +1,47 @@
+# Dictionary cache 
+
+After Percona XtraBackup takes the [backup](./how-xtrabackup-works.md#back-up) phase, in the [prepare](./how-xtrabackup-works.md#prepare) phase, `InnoDB` performs the crash recovery operation, as if the server exited.
```

Comment:
```suggestion
After Percona XtraBackup takes the [backup](./how-xtrabackup-works.md#back-up), in the [prepare](./how-xtrabackup-works.md#prepare) phase, `InnoDB` performs the crash recovery operation as if the server exited.
```

Reply:
Let's rewrite this a bit. 

A backup taken Percona XtraBackup must be prepared before being restored. The prepare phase in xtrabackup is equivalent to the InnoDB engine performing a crash recovery.

Reply:
I also see that we have somethign similar written below (how-xtrabackup works.md)

This is good too. we can use this as is

```
Percona XtraBackup is based on how [`crash recovery`](https://dev.mysql.com/doc/refman/8.0/en/glossary.html#glos_crash_recovery) works. Percona XtraBackup copies the InnoDB data files, which results in data that is internally inconsistent; then performs crash recovery on the files to make a consistent, usable database again.
```

we can say the prepare phase performs the crash recovery and make the database consistent.

Reply:
Rewritten